grease/src/Grease/Shape.hs

```diff
@@ -28,29 +29,52 @@ module Grease.Shape (
   minimalShapeWithPtrs',
   traverseShapeWithType,
   tagWithType,
+  ParsedShapes (..),
```

Comment:
FYI you can split the Haddocks into sections the same way you've done in the module itself like so:
```suggestion
  -- * Replacing
  ParsedShapes (..),
```